### PR TITLE
Add Jolokia agents to ZK instances

### PR DESCRIPTION
**Pull request by http-418**
Zookeeper should also have Jolokia agents, primarily for use on Kafka clusters lacking graphite + jmxtrans.

### CODE REVIEW

File: cookbooks/bcpc-hadoop/templates/default/zk_zookeeper-env.sh.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@ export ZOO_PID_DIR=/var/run/zookeeper
 export ZOOPIDFILE=$ZOO_PID_DIR/zookeeper-server
 export ZOOCFGDIR=<%= node[:bcpc][:hadoop][:zookeeper][:conf_dir] %>
 export ZOO_DATADIR_AUTOCREATE_DISABLE=1
-SERVER_JVMFLAGS="$SERVER_JVMFLAGS -Djute.maxbuffer=<%= node[:bcpc][:hadoop][:jute][:maxbuffer]  %>"
+SERVER_JVMFLAGS="$SERVER_JVMFLAGS -Djute.maxbuffer=<%= node[:bcpc][:hadoop][:jute][:maxbuffer]  %> <%= node[:bcpc][:jolokia][:jvm_args] %>"
```
Comment (vt0r) [inline cookbooks/bcpc-hadoop/templates/default/zk_zookeeper-env.sh.erb]: FC001 - Use strings in preference to symbols to access node attributes

(sorry)

**Comment by http-418** [comment]
I'm not sure replacing symbols with strings was a real constructive use of time but it's done now.

**Comment by cbaenziger** [comment]
Thank you for respecting our [contributing](https://github.com/bloomberg/chef-bach/blob/master/CONTRIBUTING.md) guidelines @http-418. We look forward to future commits as we all try to clean up our codebase!

**Comment by vt0r** [comment]
I assume this has already been tested?

**Comment by http-418** [comment]
Before making changes for FC and RC, yes.

**Comment by http-418** [comment]
Now I suppose I had better try it again.

**Pull request merged**